examples/Polls API.md

```diff
@@ -0,0 +1,179 @@
+FORMAT: 1A
+
+# Polls
+
+Polls is a simple web service that allows consumers to view polls and vote in them.
+
+# Polls API Root [/]
+
+This resource does not have any attributes. Instead it offers the initial API affordances in the form of the links in the JSON body.
+
+It is recommend to follow the “url” link values or Link headers to get to resources instead of constructing your own URLs to keep your client decoupled from implementation details.
```

Comment:
Perhaps instead the following, which mentions location headers. But this is perhaps too much information.

> Rather than constructing your own URLs, it is recommended that you following links exposed in the API instead to keep your client decoupled from implementation details. These links will be exposed in `url` link values in response bodies, along with [`Link`](https://tools.ietf.org/html/rfc5988) or [`Location`](http://en.wikipedia.org/wiki/HTTP_location) headers.